app/assets/javascripts/custom.js

```diff
@@ -7,40 +7,43 @@ $(document).ready(function() {
                         github_url: $("#game-github").val(),
                         screenshot_or_gif: $("#game-screenshot").val(),
                         }
-                      }
+                      };
     createGame(gameParams);
   });
 
   $("#update-game").on('click', function () {
-    var gameId = $(this).parent().parent().parent().data("game-id")
+    var gameId = $(this).parent().parent().parent().find(".game-show").data("game-id");
```

Comment:
Is there a better way to bubble this event listener a little better--less chaining somehow? Without seeing how the HTML is structured and where you're storing the data attribute, this may be your best option.